sympy/matrices/tests/test_matrices.py

```diff
@@ -1029,56 +1029,6 @@ def test_definite():
     assert m.is_indefinite == True
 
 
-def test_positive_definite():
-    # Test alternative algorithms for testing positive definitiveness.
-    m = Matrix([[2, -1, 0], [-1, 2, -1], [0, -1, 2]])
-    assert m._eval_is_positive_definite(method='eigen') == True
-    assert m._eval_is_positive_definite(method='LDL') == True
-    assert m._eval_is_positive_definite(method='CH') == True
```

Comment:
Are these tests not needed any more?

Reply:
I have removed those methods so they cannot be tested